### PR TITLE
Fix parameter name in manageChannelMembers

### PR DESCRIPTION
**Pull request by marcin-cebo**
feat: Added membershipType to Memberships and ChannelMembers APIs.

Added also a way to specify optional data being added to the response for Membership and ChannelMembers APIs.

### CODE REVIEW

File: pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt [hunk 1]
```diff
@@ -1886,7 +1886,7 @@ actual interface PubNub : StatusEmitter, EventEmitter {
     fun manageChannelMembers(
         channel: String,
         usersToSet: Collection<MemberInput>,
-        usersToRemove: Collection<String>,
+        usersIdsToRemove: Collection<String>,
```
Comment (wkal-pubnub) [inline pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt:1889]:
```suggestion
        userIdsToRemove: Collection<String>,
```

Comment (marcin-cebo) [reply pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt:1889]: Good catch. I will change it.

**Review by wkal-pubnub — APPROVED** [review]

**Comment by marcin-cebo** [comment]
@pubnub-release-bot release kotlin as v10.3.1

**Comment by marcin-cebo** [comment]
@pubnub-release-bot release kotlin as v10.3.0

**Pull request merged**